docs/resource_management/RESOURCE_SCALING.md

```diff
@@ -0,0 +1,61 @@
+# Product scaling
+For optimum performance and stability the appropriate resource `request` for each pod, and the number of pods in the product cluster should be carefully considered. Kubernetes provides a means for horizontally and vertically scaling the pods within a cluster, these approaches are described below.
+
+## Horizontal scaling
+The Helm charts provision one `StatefulSet` by default. In order to horizontally scale up or down the cluster `kubectl scale` can be used at runtime to provision a multi-node Data Center cluster, with no further configuration required (although note that the Ingress must support cookie-based session affinity in order for the products to work correctly in a multi-node configuration). Here is the syntax for scaling up/down the Data Center cluster:
+```
+kubectl scale statefulsets <statefulsetset-name> --replicas=n
+```
+For details on the `cpu` and `memory` requirements see [Resource requests and limits](#Resource-requests-and-limits) 
+
+## Vertical scaling
+The resource `requests` and `limits` for a `StatefulSet` can be defined either post product deployment or for deployments that are already running within the Kubernetes cluster. Take note that vertical scaling will result in the pod being re-created with the updated values
```

Comment:
`either post product deployment or for deployments that are already running` - that is the same thing, it should be probably something like:
`either before product deployment or for deployments that are already running`

Reply:
Also period after 'updated values'